resolver/contentResolver.go

```diff
@@ -0,0 +1,70 @@
+package resolver
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/Financial-Times/transactionid-utils-go"
+	"io/ioutil"
+	"net/http"
+	"regexp"
+)
+
+var serviceNameRegex = regexp.MustCompile("__(.*?)/")
+
+type contentResolver struct {
+	contentResolverAppURI  string
+	contentResolverAppName string
```

Comment:
In the end we agreed that it would be redundant anyway to have both URL and App Name in logs so based on the concern with k8s we will just drop the App Name for now.